css/types/length.json

```diff
@@ -438,8 +438,8 @@
               }
             },
             "status": {
-              "experimental": false,
-              "standard_track": true,
+              "experimental": true,
```

Comment:
@dipikabh FYI this is better. Note though that normally we don't record things in BCD unless there is some implementation, so (even though this predates you) @queengooborg may suggest you remove the whole entry.